packages/p/primereact.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "primereact",
+  "description": "[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![npm version](https://badge.fury.io/js/primereact.svg)](https://badge.fury.io/js/primereact)",
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "homepage": "http:///primereact/showcase",
```

Comment:
Add manual homepage

Reply:
bot should ensure that

Reply:
Yeah, just validate that its at least a URL :)